addons/compositions/functions/fnc_openDisplay.sqf

```diff
@@ -36,12 +36,9 @@ _ctrlList ctrlAddEventHandler ["LBSelChanged", {
 private _categories = [];
 
 {
-    _x params ["_category"];
-
-    if (_categories pushBackUnique _category != -1) then {
-        _ctrlList lbAdd _category;
-    };
-} forEach GET_COMPOSITIONS;
+    _categories pushBack _x;
+    _ctrlList lbAdd _x;
+} forEach keys GET_COMPOSITIONS;
```

Comment:
Don't need `_categories` variable anymore. It was only used to prevent duplicate entries from being added.